packages/app/package.json

```diff
@@ -26,6 +26,7 @@
     "@backstage/plugin-scaffolder": "^0.1.1-alpha.23",
     "@backstage/plugin-sentry": "^0.1.1-alpha.23",
     "@backstage/plugin-tech-radar": "^0.1.1-alpha.23",
+    "@backstage/plugin-tech-stack": "^0.1.1-alpha.23",
```

Comment:
Please also remove it from here.

Reply:
No, not at all! Don't get me wrong, I meant that your changes should essentially all be inside `plugins/tech-stack`